src/wrappers/React.js

```diff
@@ -10,11 +10,13 @@ const makeReactContainer = Component => {
       super(props)
 
       /**
-       * We create a stateful component in order to attach a ref on it. We will use that ref to
-       * update component's state, which seems better than re-rendering the whole thing with
+       * We previously created a stateful component in order to attach a ref on it. We have used that ref to
```

Comment:
Actually, lines 13 and 14 shouldn't be changed. Comments there are not related to the issue.